packages/mds-jurisdiction-service/tests/index.spec.ts

```diff
@@ -106,25 +106,71 @@ describe('Write/Read Jurisdictions', () => {
     test.value(jurisdiction).is(null)
   })
 
+  it('Update One Jurisdiction (invalid jurisdiction_id)', async () => {
+    const [error, jurisdiction] = await JurisdictionService.updateJurisdiction(JURISDICTION_ID, {
+      jurisdiction_id: uuid(),
+      agency_name: 'Some New Name',
+      timestamp: TODAY
+    })
+    test
+      .value(error)
+      .isNot(null)
+      .isInstanceOf(ValidationError)
+    test.value(jurisdiction).is(null)
+  })
+
+  it('Update One Jurisdiction (invalid timestamp)', async () => {
+    const [error, jurisdiction] = await JurisdictionService.updateJurisdiction(JURISDICTION_ID, {
+      agency_name: 'Some New Name',
+      timestamp: LAST_WEEK
+    })
+    test
+      .value(error)
+      .isNot(null)
+      .isInstanceOf(ValidationError)
+    test.value(jurisdiction).is(null)
+  })
+
+  it('Update One Jurisdiction (not found)', async () => {
+    const [error, jurisdiction] = await JurisdictionService.updateJurisdiction(uuid(), {
+      agency_name: 'Some New Name',
+      timestamp: TODAY
+    })
+    test
+      .value(error)
+      .isNot(null)
+      .isInstanceOf(NotFoundError)
+    test.value(jurisdiction).is(null)
+  })
+
+  it('Update One Jurisdiction', async () => {
+    const [error, jurisdiction] = await JurisdictionService.updateJurisdiction(JURISDICTION_ID, {
+      agency_name: 'Some New Name',
+      timestamp: TODAY
+    })
+    test.value(jurisdiction).isNot(null)
+    test.value(jurisdiction?.jurisdiction_id).is(JURISDICTION_ID)
+    test.value(jurisdiction?.timestamp).is(TODAY)
+    test.value(error).is(null)
+  })
+
   it('Read Specific Jurisdiction (current version)', async () => {
     const [error, jurisdiction] = await JurisdictionService.getOneJurisdiction(JURISDICTION_ID)
     test.value(jurisdiction).isNot(null)
     test.value(jurisdiction?.jurisdiction_id).is(JURISDICTION_ID)
-    test.value(jurisdiction?.timestamp).is(YESTERDAY)
+    test.value(jurisdiction?.timestamp).is(TODAY)
     test.value(error).is(null)
   })
 
-  // Requires updateJurisdiction
-  //
-  // it('Read Specific Jurisdiction (prior version)', async () => {
-  //   const [error, jurisdiction] = await JurisdictionService.getOneJurisdiction(JURISDICTION_ID, {
-  //     effective: YESTERDAY
-  //   })
-  //   test.value(jurisdiction).isNot(null)
-  //   test.value(jurisdiction?.jurisdiction_id).is(JURISDICTION_ID)
-  //   test.value(jurisdiction?.timestamp).is(YESTERDAY)
-  //   test.value(error).is(null)
-  // })
+  it('Read Specific Jurisdiction (prior version)', async () => {
+    const [error, jurisdiction] = await JurisdictionService.getOneJurisdiction(JURISDICTION_ID, {
+      effective: YESTERDAY
```

Comment:
I don't understand why you'd ever want to pass in the timestamp. Is a `jurisdiction_id` not uniquely identifying?

Reply:
It is uniquely identifying and has to be ... however, jurisdictions can have versions that change over time.

Reply:
Never mind, read the spec.